tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
```

Comment:
@VortexExpansion 
Can we remove this?

Reply:
@yabe-diverta san

Refactored to jsconfig.json https://github.com/diverta/front_next_media/pull/5/commits/5d963366a47a493354307854d7fe85e10cd4a766

Please review. 🙇

Reply:
looks good, thank you 🙇